tests/Observatory/test_ObservatoryL2Halo.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+from tests.TestSupport.Info import resource_path
+import EXOSIMS.Observatory.ObservatoryL2Halo as obs
+import pkgutil, os, json, sys, copy
+from EXOSIMS.util.get_module import get_module
+import numpy as np
+from astropy.time import Time
+import astropy.units as u
+from tests.TestSupport.Utilities import RedirectStreams
+
+# Python 3 compatibility:
+if sys.version_info[0] > 2:
```

Comment:
We are removing these Python2-Python3 compatability code blocks when possible. We only support python3 at the moment.